server/repository/src/db_diesel/sync_file_reference_row.rs

```diff
@@ -0,0 +1,85 @@
+use super::sync_file_reference_row::sync_file_reference::dsl::*;
+
+use crate::RepositoryError;
+use crate::StorageConnection;
+
+use chrono::NaiveDateTime;
+use diesel::prelude::*;
+use serde::{Deserialize, Serialize};
+
+table! {
+    sync_file_reference (id) {
+        id -> Text,
+        table_name -> Text,
+        record_id -> Text,
+        file_name -> Text,
+        created_datetime -> Timestamp,
+        deleted_datetime -> Nullable<Timestamp>,
+    }
+}
+
+#[derive(
+    Clone, Insertable, Queryable, Debug, PartialEq, AsChangeset, Eq, Default, Serialize, Deserialize,
+)]
+#[table_name = "sync_file_reference"]
+pub struct SyncFileReferenceRow {
+    pub id: String,
+    pub table_name: String, // Maybe should be ChangeLogTableName ? But that causes mapping problems in the api request...
```

Comment:
table_name is fine by me - I'm not concerned at having it not reference the changelog here